Character.cpp

```diff
@@ -87,31 +89,44 @@ int Character::takeDamage(int damage)
 }
 
 
-#include <assert>
+//#include <assert>
 void Character::attackInternal(Character& other)
 {
     if( other.hitPoints <= 0 )
-    {
+    {   
+        *initialHitPoints *= 1.1;
+
+        if (hitPoints < *initialHitPoints)
+        {
+            hitPoints = *initialHitPoints;
+        }
+
+        *initialArmorLevel *= 1.1;
+
+        if (armor < *initialArmorLevel)
+        {
+            armor = *initialArmorLevel;
+        }
+
```

Comment:
There's a 3rd attribute you didn't update.
Your logic does not follow a) b) c)
once you figure out the correct logic, you need to D.R.Y. refactor it.  use a free function.
- [ ] revise.

Reply:
Think its good